euth_wagtail/assets/scss/components/_badges.scss

```diff
@@ -0,0 +1,28 @@
+.euth-badge {
+    @extend .badge;
+    border-radius: 100px;
+    @include rem(padding, 4px 20px);
+    color: #fff;
+    font-size: 12px;
+    font-weight: bold;
+    font-style: italic;
+    line-height: 2;
+    letter-spacing: -0.2px;
```

Comment:
Please check what styling you can do by overriding the `bootstrap-variables.scss` first. https://github.com/liqd/euth_wagtail/blob/master/euth_wagtail/assets/scss/core/_bootstrap-variables.scss